protocol/x/clob/keeper/liquidations.go

```diff
@@ -452,6 +452,24 @@ func (k Keeper) GetFillablePrice(
 		return nil, err
 	}
 
+	// stat liquidation order for negative TNC when in DeliverTx
+	if tncBig.Sign() < 0 && !ctx.IsCheckTx() && !ctx.IsReCheckTx() {
+		telemetry.IncrCounterWithLabels(
+			[]string{metrics.Liquidations, metrics.LiquidationMatchNegativeTNC, metrics.DeliverTx},
+			1,
+			[]gometrics.Label{
+				metrics.GetLabelForStringValue(
+					metrics.SubaccountOwner,
+					subaccountId.Owner,
+				),
```

Comment:
IMO if you want to leave this metric label as-is, would recommend one of the following approaches:
- Add a comment noting that metric is infinite cardinality but it was left here because it's useful for observability.
- Emit an info log instead.

I'm leaning towards 2nd approach but no strong opinion.

Reply:
will update to info log